content/en/tracing/profiler/enabling/ddprof.md

```diff
@@ -14,34 +16,45 @@ further_reading:
 ---
 
 <div class="alert alert-warning">
-The Datadog Profiler for Linux is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+The compiled languages profiler, <code>ddprof</code> is in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
 </div>
 
-The Datadog Profiler for Linux lets you collect profile data for applications running on Linux. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust. It can also profile services such as nginx, Redis, or Postgres, as it does not require access to source code.
+The Datadog Profiler for compiled languages (`ddprof`) uses OS level APIs to collect profiling data. It is ideally suited for applications written in compiled languages, such as C, C++, or Rust.
 
 ## Requirements
 
-The Datadog Profiler for Linux is compatible with Linux v4.17+ on `amd64` or `arm64` processors. It is not supported on serverless platforms, such as AWS Lambda, or on operating systems other than Linux, such as Windows, BSD, and MacOS.
+Supported operating systems
+: Linux kernel v4.17+, libc 2.23+
```

Comment:
Looking at the current version of ddprof, we actually support glibc 2.18+.

For the curious:
```
$ readelf -sW ./ddprof | grep GLIBC_ | sed 's/.*GLIBC/GLIBC/g' | sort | uniq
GLIBC_2.10
GLIBC_2.10 (11)
GLIBC_2.14
GLIBC_2.14 (4)
GLIBC_2.15
GLIBC_2.15 (18)
GLIBC_2.16
GLIBC_2.16 (14)
GLIBC_2.18
GLIBC_2.18 (17)
<snip>
```

Reply:
We could indeed add a test in the CI / releasing to make sure we never exceed 2,18. For now the guarantee was around 2.23 (due to build infra). Considering you validated this for the current release, it is worth mentioning it.

Reply:
Too bad we don't naturally have centos 7 covered ! (2.17 afaik)